test/CadlRanchProjects/client/naming/src/Generated/NamingClient.cs

```diff
@@ -180,30 +180,30 @@ public virtual Response Parameter(string clientName, RequestContext context = nu
         }
 
         /// <summary> Client. </summary>
-        /// <param name="clientNameModel"> The <see cref="ClientNameModel"/> to use. </param>
+        /// <param name="body"> The <see cref="ClientNameModel"/> to use. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="clientNameModel"/> is null. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="body"/> is null. </exception>
         /// <include file="Docs/NamingClient.xml" path="doc/members/member[@name='ClientAsync(ClientNameModel,CancellationToken)']/*" />
-        public virtual async Task<Response> ClientAsync(ClientNameModel clientNameModel, CancellationToken cancellationToken = default)
```

Comment:
This is due to the change in `cadl-ranch`: https://github.com/Azure/cadl-ranch/commit/fd61404ddd3d374be328af588b11985af4869189